.travis.yml

```diff
@@ -3,12 +3,15 @@ sudo: false
 notifications:
   email: false
 
+before_install:
+  - yes | sdkmanager "platforms;android-28"
+
 script: ./CI/travis.sh
 
 android:
   components:
     - tools
-    - build-tools-26.0.2
+    - build-tools-28.0.3
     - platform-tools
     - tools
-    - android-27
+    - android-28
```

Comment:
Is it okay for us to update to version 28 of the API?

Reply:
You're the one who bumped it. You should know. :)

Reply:
Not exactly, as I believe xyzz used a build tool outside of Android API 27.